tests/testthat/test-expressions.R

```diff
@@ -17,6 +17,14 @@ test_that(".dispatchFilter uses right numeric function", {
 with_mock_crunch({
     ds <- loadDataset("test ds")
 
+    test_that("is method works for both expressions and logical expressions", {
+        e1 <- try(ds$birthyr + 5)
+        expect_true(is.CrunchExpr(e1))
```

Comment:
just `expect_true(is.CrunchExpr(ds$birthyr + 5))`

and you can have these in the unit tests.